Jenkinsfile

```diff
@@ -498,7 +498,7 @@ stage('Non-default environments') {
 				withMavenWorkspace {
 					// Re-run integration tests against the JARs produced by the default build,
 					// but using a different JDK to build and run the tests.
-					mavenNonDefaultBuild buildEnv, "-f integrationtest"
+					mavenNonDefaultBuild buildEnv, "-f integrationtest -Pci-build"
```

Comment:
We'll need to enable this profile basically everywhere, no? If so you could just add it inside `mavenNonDefaultBuild`.

And you probably want to enable it in the default build as well?

Reply:
Or, I guess, we could just set the testcontainers environment variable in Jenkinsfiles instead of using Maven.

Reply:
we don't need it for compile-check sources (pre-build step) and for eclipse compiler case ... but then I guess it doesn't hurt to have it for them too 😃